include/rtc/rtc.h

```diff
@@ -176,6 +199,113 @@ RTC_EXPORT int rtcDeleteTrack(int tr);
 
 RTC_EXPORT int rtcGetTrackDescription(int tr, char *buffer, int size);
 
+// Media
+#if RTC_ENABLE_MEDIA
+
+/// Add track
+/// @param pc Peer connection id
+/// @param codec Codec
+/// @param payloadType Payload type
+/// @param ssrc SSRC
+/// @param _mid MID (default: video|audio)
+/// @param _direction Direction (default RTC_SENDONLY)
+/// @param _name Name (optional)
+/// @param _msid MSID (optional)
+/// @returns Track id
+RTC_EXPORT int rtcAddTrackEx(int pc, rtcCodec codec, int payloadType, uint32_t ssrc, const char *_mid, rtcDirection * _direction, const char *_name, const char *_msid);
+
+/// Set H264PacketizationHandler for track
+/// @param tr Track id
+/// @param ssrc SSRC
+/// @param cname CName
+/// @param payloadType Payload Type
+/// @param clockRate Clock rate
+/// @param _sequenceNumber Sequence number (default: pseudo random value)
+/// @param _timestamp Timestamp (default: pseudo random value)
+RTC_EXPORT int rtcSetH264PacketizationHandler(int tr, uint32_t ssrc, const char * cname, uint8_t payloadType, uint32_t clockRate, uint16_t * _sequenceNumber, uint32_t * _timestamp);
```

Comment:
Same here for `uint16_t * _sequenceNumber` and `uint32_t * _timestamp`, could `(uint16_t)-1` be interpreted as a random value? If so, you could define `RTC_RANDOM_SEQUENCE_NUMBER` and `RTC_RANDOM_TIMESTAMP`.

Reply:
In that case I would prefer to remove "optional" and let users to choose value for sequence number and timestamp.

Reply:
OK, works for me!